config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_accesscontextmanagerserviceperimeterresources.accesscontextmanager.cnrm.cloud.google.com.yaml

```diff
@@ -24,6 +24,142 @@ spec:
   preserveUnknownFields: false
   scope: Namespaced
   versions:
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    - description: When 'True', the most recent reconcile of the resource succeeded
+      jsonPath: .status.conditions[?(@.type=='Ready')].status
+      name: Ready
+      type: string
+    - description: The reason for the value in 'Ready'
+      jsonPath: .status.conditions[?(@.type=='Ready')].reason
+      name: Status
+      type: string
+    - description: The last transition time for the value in 'Status'
+      jsonPath: .status.conditions[?(@.type=='Ready')].lastTransitionTime
+      name: Status Age
+      type: date
+    name: v1beta1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'apiVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              perimeterNameRef:
+                oneOf:
+                - not:
+                    required:
+                    - external
+                  required:
+                  - name
+                - not:
+                    anyOf:
+                    - required:
+                      - name
+                    - required:
+                      - namespace
+                  required:
+                  - external
+                properties:
+                  external:
+                    description: 'Allowed value: The `name` field of an `AccessContextManagerServicePerimeter`
```

Comment:
Ouch - I might simply not have noticed this before, but it's hard to understand that this is the GCP "short self link", rather than a KCC object name

Reply:
(Not a blocker, but maybe we can file a github issue)

Reply:
Yes, one enhancement is to mention it is the `status.name` field. Some code change will be needed.

Reply:
Created https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/1310